extensions/DT/cameracontrols.js

```diff
@@ -27,7 +27,7 @@
 
   vm.runtime.runtimeOptions.fencing = false;
   vm.renderer.offscreenTouching = true;
-  vm.setInterpolation(false);
+  vm.setInterpolation(0);
```

Comment:
dont think that's intentional?

Reply:
It is, "setInterpolation" apparently accepts number values, not booleans

Reply:
The types say so:
![image](https://github.com/TurboWarp/extensions/assets/68464103/4708ce0a-983d-4913-a117-1f82e493645d)

Reply:
the types are wrong https://github.com/TurboWarp/extensions/pull/961